README.md

```diff
@@ -0,0 +1,30 @@
+# 🛠️ Lowdefy Tutorial
+
+![lowdefy logo](https://lowdefy.com/images/logo.svg)
```

Comment:
wondering if we want to adjust the image so it doesn't take up the whole screen? If so, can use something similar to below, if not, no worries. 
`<img src="https://lowdefy.com/images/logo.svg" width="400" height="150">`

legibility on dark mode isn't great. not sure if we want a fix for this or just leave it as is for now is fine.

Reply:
afiak you cannot resize an image in markdown, you are right about dark mode, I am going to replace this with a png or jpeg with a background color

Reply:
d0152378cc26daaa1ff74ed0494ffba0862aebda